vips/processor.go

```diff
@@ -251,8 +255,11 @@ func (v *Processor) newThumbnailFallback(
 }
 
 // NewImage creates new Image from imagor.Blob
-func (v *Processor) NewImage(ctx context.Context, blob *imagor.Blob, n, page int) (*Image, error) {
+func (v *Processor) NewImage(ctx context.Context, blob *imagor.Blob, n, page int, dpi *int) (*Image, error) {
 	var params = NewImportParams()
+	if dpi != nil {
+		params.Density.Set(*dpi)
```

Comment:
`var dpi int` and check by `dpi > 0` instead? Using pointer seems a bit overkill

Reply:
sure, done!